packages/openapi-v3/src/json-to-schema.ts

```diff
@@ -4,11 +4,12 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {JsonDefinition} from '@loopback/repository-json-schema';
-import {SchemaObject, ExtensionValue} from '@loopback/openapi-v3-types';
+import {SchemaObject} from '@loopback/openapi-v3-types';
 import * as _ from 'lodash';
 
 export function jsonToSchemaObject(jsonDef: JsonDefinition): SchemaObject {
-  const json = jsonDef as {[name: string]: ExtensionValue}; // gets around index signature error
+  // tslint:disable-next-line:no-any
+  const json = jsonDef as {[name: string]: any}; // gets around index signature error
```

Comment:
hmm, any reason changing `ExtensionValue` to `any`? I thought this PR aims at importing `ExtensionValue` from `openapi-spec-builder`.
Not a big deal though.

Reply:
JSON Definition is a JSON Schema object containing well-known keys and values:

```ts
export interface JsonDefinition extends Definition {
  allOf?: JsonDefinition[];
  oneOf?: JsonDefinition[];
  anyOf?: JsonDefinition[];
  items?: JsonDefinition | JsonDefinition[];
  additionalItems?: {
    anyOf: JsonDefinition[];
  };
  enum?: PrimitiveType[] | JsonDefinition[];
  additionalProperties?: JsonDefinition | boolean;
  definitions?: {[definition: string]: JsonDefinition};
  properties?: {[property: string]: JsonDefinition};
}
```

There are not OpenAPI spec extensions involved AFAICT, therefore we should not be using `ExtensionValue` type.

Based on the comment in the code, my understanding is that we need to cast from `JsonDefinition` to a string-key-to-any-value object/map to fix a compiler error in the `default` case below:

```ts
default:
  result[property] = json[property];
  break;
```

I found a more elegant way how solve this problem by casting `property` to `keyof JsonDefinition`, see 8de9b75f3.